docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/link/link.scss

```diff
@@ -51,6 +51,18 @@
     }
   }
 
+  &.civictheme-link--external {
+    &::after {
```

Comment:
please extract this into `civictheme-icon-inline()` mixin and allow to pass content as a parameter. Then, use this mixin here with a `$civictheme-link-icon-external;` variable

Please rename `$civictheme-link-external` to `$civictheme-link-icon-external`